brion/blueConfig.h

```diff
@@ -74,6 +74,12 @@ class BlueConfig : public boost::noncopyable
                                       const std::string& sectionName,
                                       const std::string& key ) const;
 
+    /**
+     * @return the user and circuit file target parsers.
+     * @version 1.7
+     */
+    BRION_API Targets getTargetParsers() const;
```

Comment:
I thought we had agreed that brain::Circuit won't store a BlueConfig, but will copy over the information that it needs and this is one of this pieces of information in order to implement Circuit::getGIDs().
I can store a BlueConfig in brain::Circuit but then the constructor which takes the const BlueConfig& (which I find convenient) must go away unless BlueConfig is modified because it cannot be copied and there's no way to retrieve the path used at construction to create a new one.

Given that brion::BlueConfig is the low level access layer, I don't think it's a bad idea to have this method here.